CHANGELOG.md

```diff
@@ -7,3 +7,17 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 ## Added
 - Basic Skel
 - missing PR template
+
+## [0.0.2] - 2016-08-11
```

Comment:
Lets drop this, I know this came from obviously the fact that you started this before we created this repo. From my perspective we are `0.0.1`